.circleci/config.yml

```diff
@@ -25,11 +25,17 @@ references:
         curl -X POST -H 'Content-Type: application/json' ${ROCKET_HOOK_URL:?} --data "${HOOK_DATA:?}"
       when: on_fail
 
+  setup_ci_environment: &setup_ci_environment
+    run:
+      name: Setup Environment
+      command: echo "source /infrastructure/import-secrets.sh" >> $BASH_ENV
```

Comment:
Should this not have been run by the Docker entrypoint, hence be available in following packer step, hence making this added step unnecessary? I see nothing before this step that may have changed outcome of script so this seems to me to only have the effect of writing BASH_ENV file as opposed to rely on already-set environment to be accessible to step. Could you please clarify?

Reply:
CircleCI does not run the entrypoints by default. There are options, but I'll try to improve that in future, now I strive to make it work and stabilize the configuration after the Vault integration damage :)